compiler/rustc_metadata/src/creader.rs

```diff
@@ -1171,12 +1206,18 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
                     }
                     None => item.ident.name,
                 };
+
                 let dep_kind = if attr::contains_name(&item.attrs, sym::no_link) {
                     CrateDepKind::MacrosOnly
                 } else {
                     CrateDepKind::Explicit
                 };
 
+                if name == sym::compiler_builtins {
+                    info!("BUILTINS DETECTED dep_kind {dep_kind:?}");
+                    return None;
```

Comment:
Why are you ignoring explicit `extern crate compiler_builtins;`? I think this is the reason of the CI failure as the implicitly injected compiler_builtins dependency isn't checked for stability, yet explicit mentions should be.

Reply:
Oh, that's dumb. I did this as a more minimal step than removing it from `standard_library_imports` for experimentation, but you're right that this would catch the explicit extern as well.

Reply:
Thanks, that was it. I updated https://github.com/rust-lang/rust/pull/136226 with the changes here, so will close this one.